requirements-ci.txt

```diff
@@ -1,2 +1,3 @@
 -r requirements-dev.txt
--r ./compiler_opt/benchmark/requirements.txt
\ No newline at end of file
+-r ./compiler_opt/benchmark/requirements.txt
+-r ./compiler_opt/tools/requirements.txt
```

Comment:
so I understand: it's not like the CI actually needs this right now, correct?

Reply:
The CI needs it for this patch or otherwise the `pytype` check will fail because it won't be able to import `shap` when checking the files that include it.

Reply:
aa! thanks.